app.js

```diff
@@ -90,6 +106,17 @@ app.get('/components*', function (req, res) {
   } else {
     // Show the isolated component preview
     path = path.replace(/preview/g, '')
```

Comment:
Why does this happen three times within this else block? Can it not just be done once, e.g. at the end?